javascript/src/detectors/javascript-jwt-exposed-credential/javascript-jwt-exposed-credential_compliant.js

```diff
@@ -0,0 +1,17 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+// {fact rule=javascript-jwt-exposed-credential@v1.0 defects=0}
+const jwt = require('jsonwebtoken');
+
+const payload = {
+    // Compliant: Contains only non-sensitive data (no passwords or sensitive info).
```

Comment:
Fix for the following comment by @grl-wang 
> miss compliant comment and example look like Noncompliant 
Ref: #91 

Change made => Added the appropriate comment which explains that sensitive information like "password" is not a part of the payload.